crates/sui-indexer/src/types.rs

```diff
@@ -161,6 +160,7 @@ impl IndexedEpochInfo {
             protocol_version: 0,
             total_stake: 0,
             storage_fund_balance: 0,
+            system_state: vec![],
```

Comment:
can we have a note here about the choice of 1. not overriding with the next epoch state and 2. deciding to tolerate missing some in-epoch changes ?

Reply:
`system_state_summary` arg seems no longer needed?